cmd/hw_06/task_02/server.go

```
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+	"unicode/utf8"
+)
+
+func main() {
+
+	fmt.Println("Launching server...")
+	ln, _ := net.Listen("tcp", ":8081")
+	conn, _ := ln.Accept()
+
+	for {
+		message, _ := bufio.NewReader(conn).ReadString('\n')
```

Don't ignore errors!! You get the panic when clien exits :)

			message, err := bufio.NewReader(conn).ReadString('\n')
			if err != nil {
				fmt.Println(err)
				break
			}